app/src/main/java/com/keylesspalace/tusky/util/ThemeUtils.java

```diff
@@ -121,12 +121,7 @@ public static void setAppNightMode(String flavor, Context context) {
                 break;
         }
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
Why did you remove the whole if instead of something like this:

```
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
            UiModeManager uiModeManager = (UiModeManager)context.getApplicationContext().getSystemService(Context.UI_MODE_SERVICE);
            uiModeManager.setNightMode(mode);
        }

        AppCompatDelegate.setDefaultNightMode(mode);
```